catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ExportCommand.java

```diff
@@ -143,20 +144,20 @@ protected Object executeWithSubject() throws Exception {
         final File outputFile = initOutputFile(output);
         if (outputFile.exists()) {
             printErrorMessage(String.format("File [%s] already exists!", outputFile.getPath()));
-            return null;
```

Comment:
Why this change? In all our other commands, we simply return after printing the error message. What advantage are you gaining from the runtime exception?

Is this change here just to support unit tests?

What is the effect on the runtime system?